File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.js
function solution(A, B) {
    A.sort((a, b) => b - a);
    B.sort((a, b) => b - a);
    
    let answer = 0;
    let bIdx = 0;
    for(let aIdx = 0; aIdx < A.length; aIdx++) {
        if(A[aIdx] < B[bIdx]) {
            answer++;
            bIdx++;
        }
    }
    
    return answer;
}

File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.py
def solution(A, B):
    A.sort(reverse=True)
    B.sort(reverse=True)
    
    answer = 0
    b_idx = 0
    for i in range(len(A)):
        if A[i] < B[b_idx]:
            answer += 1
            b_idx += 1
    
    return answer